### ISSUE TITLE
Change text of SvgTextSpan before Render fails

### ISSUE THREAD

**Issue by ravjak** [vvvv/SVG#125]
Hi,
Has anyone tried to change text of SvgTextSpan dynamically? I'm using SvgViewer App as a test field. In open_Click function I add a search code for a uniq id and try replace it with some magic string "mariola"

``` c#
SvgDocument svgDoc = SvgDocument.Open(openSvgFile.FileName);
SvgTextSpan tspan = svgDoc.GetElementById<SvgTextSpan>("tspan9936-4");
if (tspan != null)
{
    tspan.Text = "mariola";
}
RenderSvg(svgDoc);
```

I saw text change in debugger but it renders text previously read in open function.
I know that it is possible to parse svg as xml, change some values in it and use SvgDocument::Open<T>(Stream stream). But this does not seems to be an elegant way.

**Comment by ravjak** [comment]
I have found the problem in Text setter. I had to add code for setting it's content for all content nodes. I do not know if this is good solution but it works in my case.
```c#
public virtual string Text
{
	get {return base.Content;}
	set
	{           
		foreach (SvgContentNode node in this.GetContentNodes())
		{
			if (node.Content == this.Content)
				node.Content = value;
		}
		base.Content = value; this.IsPathDirty = true; this.Content = value;
	}
}
```

**Comment by avogelba** [comment]
with this I get: 

> Unable to cast object of type 'Svg.SvgTextSpan' to type 'Svg.SvgContentNode'.

**Comment by mrbean-bremen** [comment]
Looks like that has been fixed with #262 - closing.
